resources/schema.json

```diff
@@ -473,7 +553,7 @@
               },
               "pattern": {
                 "$id": "#/properties/property/constraints/pattern",
-                
+
```

Comment:
Accident?

Reply:
We don't make mistakes, just happy little accidents.

Fixed